level_1/new_id_recommendation.py

```diff
@@ -0,0 +1,63 @@
+# 신규 아이디 추천 문제
+# n : 문자열의 길이(length(new_id))
+# 시간복잡도 : O(n), 공간 복잡도: O(n)
+import re
+
+
+def step_four(new_id: str) -> str:
+    start, end = 0, len(new_id)
+    # 문자열이 . 만 남았을 시 빈 문자열을 반환
+    if new_id == '.':
+        return ''
+    # 문자열 맨 앞의 .을 제거
+    if new_id[0] == '.':
+        start += 1
+    # 문자열 맨 끝의 .을 제거
+    if new_id[-1] == '.':
+        end -= 1
+    return new_id[start: end]
+
+
+def step_five(new_id: str) -> str:
+    # new_id 가 빈 문자열일 경우 a 반환
+    if new_id == '':
+        return 'a'
+    return new_id
```

Comment:
```suggestion
    return new_id if new_id != '' else 'a'
```
if else 한 줄로 줄이는 거 추천!!